packages/react/src/components/Tooltip/Tooltip.test.tsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import userEvent from '@testing-library/user-event';
+import { act, render as renderRtl, screen } from '@testing-library/react';
+
+import type { TooltipProps } from './Tooltip';
+import { Tooltip } from './Tooltip';
+
+const render = async (props: Partial<TooltipProps> = {}) => {
+  const allProps: TooltipProps = {
+    children: <button>My button</button>,
+    content: 'Tooltip text',
+    ...props,
+  };
+  renderRtl(<Tooltip {...allProps} />);
+  /* Flush microtasks */
+  await act(async () => {});
+};
+
+const user = userEvent.setup();

```

Comment:
This should be inside your test or part of your `render` function in this use-case as per recommendation: https://testing-library.com/docs/user-event/intro#writing-tests-with-userevent